SMLHelper.Tests/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,20 @@
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+[assembly: AssemblyTitle("SMLHelper.Tests")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("SMLHelper.Tests")]
+[assembly: AssemblyCopyright("Copyright ©  2019")]
+[assembly: AssemblyTrademark("")]
+[assembly: AssemblyCulture("")]
+
+[assembly: ComVisible(false)]
+
+[assembly: Guid("ebc4aaf3-a13b-427e-bf99-3ae23edeb4b8")]
```

Comment:
Guid is not required if ComVisible is false

Reply:
This is auto-generated by Visual Studio and is ultimately harmless.